test/jdk/java/io/FileOutputStream/ManyFiles.java

```diff
@@ -24,6 +24,13 @@
 /* @test
  * @bug 4189011 5019303
  * @summary Test opening over 2048 files
+ * @comment Vestigial comment from before the requires tag was added:
+ * Linux does not yet allow opening this many files; Solaris
+ * 8 requires an explicit allocation of more file descriptors
+ * to succeed. Since this test is written to check for a
+ * Windows capability it is much simpler to only run it
+ * on that platform.
+ * @requires (os.family != "linux")
```

Comment:
I think this comment should be replaced with a clear summary as to why three is a requires. No need to mention Solaris.

Reply:
Perhaps the `@requires` tag is no longer needed. Without it, this test passed 30 repeats in the CI on both Intel and ARM Linux nodes.

Reply:
Comment simplified in 4cee3c2.

Reply:
Okay, just a bit puzzled as to why it doesn't run on Linux. The VM uses setrlimit(RLIMIT_NOFILE) to set the limit to the hard limit (rlim_max). So is the issue here that the hard limit less than 2050 on some test systems?